client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -157,7 +157,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 				buttonLabel={ translate( 'Add a business address' ) }
 				e2eType="business-address"
 				header={ this.renderHeader() }
-				illustration="/calypso/images/illustrations/business-address.svg"
+				illustration="/calypso/images/illustrations/illustration-businessaddress-jetpack.svg"
```

Comment:
I'd name this one `jetpack-business-address.svg` instead, or even  (`business-address.svg` - we introduced this illustration specifically for this step, so we can replace it).

Reply:
👍